src/test/java/CalendarServletTest.java

```diff
@@ -105,40 +116,49 @@ public void setUp() throws IOException, GeneralSecurityException {
   @Test
   public void noCalendarEvent() throws IOException, ServletException {
     // Test case where there are no events in the user's calendar
-    Mockito.when(calendarClient.getCalendarEvents()).thenReturn(NO_EVENT);
-    checkReturn(NO_EVENT);
+    Mockito.when(calendarClient.getCalendarList()).thenReturn(ONE_CALENDAR);
+    Mockito.when(calendarClient.getCalendarEvents(PRIMARY)).thenReturn(NO_EVENT);
+    List<Event> actual = checkReturn();
+    Assert.assertEquals(NO_EVENT, actual);
   }
 
   @Test
-  public void firstTwoEvents() throws IOException, ServletException {
-    // Test case where there are two events with defined summaries
-    // Events must be returned in order of retrieval - JSON includes tasks in desired order
-    Mockito.when(calendarClient.getCalendarEvents()).thenReturn(EVENT_ONE_TWO);
-    checkReturn(EVENT_ONE_TWO);
+  public void twoCalendars() throws IOException, ServletException {
+    // Test case where there are two calendars with a defined event in each
+    // Events must be returned in order of retrieval
+    Mockito.when(calendarClient.getCalendarList()).thenReturn(TWO_CALENDARS);
+    Mockito.when(calendarClient.getCalendarEvents(PRIMARY)).thenReturn(EVENT_ONE);
+    Mockito.when(calendarClient.getCalendarEvents(SECONDARY)).thenReturn(EVENT_TWO);
+    List<Event> actual = checkReturn();
+    Assert.assertEquals(EVENT_ONE_TWO, actual);
   }
 
   @Test
   public void undefinedEvent() throws IOException, ServletException {
     // Test case where there is an event with no summary
-    Mockito.when(calendarClient.getCalendarEvents()).thenReturn(EVENT_UNDEFINED);
-    checkReturn(EVENT_UNDEFINED);
+    Mockito.when(calendarClient.getCalendarList()).thenReturn(ONE_CALENDAR);
+    Mockito.when(calendarClient.getCalendarEvents(PRIMARY)).thenReturn(EVENT_UNDEFINED);
+    List<Event> actual = checkReturn();
+    Assert.assertEquals(EVENT_UNDEFINED, actual);
   }
 
   @Test
   public void allEvent() throws IOException, ServletException {
     // Test case where there are two defined and an undefined event
     // Events must be returned in order of retrieval - JSON includes tasks in desired order
-    Mockito.when(calendarClient.getCalendarEvents()).thenReturn(EVENT_ALL);
-    checkReturn(EVENT_ALL);
+    Mockito.when(calendarClient.getCalendarList()).thenReturn(ONE_CALENDAR);
+    Mockito.when(calendarClient.getCalendarEvents(PRIMARY)).thenReturn(EVENT_ALL);
+    List<Event> actual = checkReturn();
+    Assert.assertEquals(EVENT_ALL, actual);
   }
 
-  public void checkReturn(List<Event> EXPECTED_EVENT) throws IOException, ServletException {
+  public List<Event> checkReturn() throws IOException, ServletException {
```

Comment:
Nit: rename this helper to something like `doGetAsJSON` or equally descriptive. (It's not checking anything now.)

(Also, it doesn't need to be public.)